.circleci/config.yml

```diff
@@ -159,20 +169,17 @@ jobs:
           key: heads-musl-cross-{{ checksum "./tmpDir/musl-cross.sha256sums" }}{{ .Environment.CACHE_VERSION }}
           paths:
             - crossgcc
-            - build/musl-cross-38e52db8358c043ae82b346a2e6e66bc86a53bc1
+            - build/*/musl-cross-*
```

Comment:
@SergiiDmytruk :

Confirmed on the saving cache steps that are happening now under https://app.circleci.com/pipelines/github/SergiiDmytruk/heads/57/workflows/dda551e0-812a-4c6a-94d6-bd59a640f0d5/jobs/780

musl-cross level 1 cache:
https://app.circleci.com/pipelines/github/SergiiDmytruk/heads/57/workflows/dda551e0-812a-4c6a-94d6-bd59a640f0d5/jobs/780/parallel-runs/0/steps/0-102:
`Warning: could not archive /root/project/build/*/musl-cross-* - Not found`

coreboot level 2 cache:
https://app.circleci.com/pipelines/github/SergiiDmytruk/heads/57/workflows/dda551e0-812a-4c6a-94d6-bd59a640f0d5/jobs/780/parallel-runs/0/steps/0-103
```
Warning: could not archive /root/project/build/*/musl-cross-* - Not found
Warning: could not archive /root/project/build/*/coreboot-* - Not found
```

level 3 cache will most probably be successful since it encompasses everything `build/` (unless it is too big... we will see):
https://app.circleci.com/pipelines/github/SergiiDmytruk/heads/57/workflows/dda551e0-812a-4c6a-94d6-bd59a640f0d5/jobs/780/parallel-runs/0/steps/0-104